measurement/plugins/speedtestdotnet/results.py

```diff
@@ -8,7 +8,6 @@
 else:
     SpeedtestdotnetMeasurementResult = collections.namedtuple(
         "SpeedtestdotnetMeasurementResult",
-        "id errors host minimum_latency average_latency maximum_latency median_deviation "
-        "packets_transmitted packets_received packets_lost packets_lost_unit time time_unit",
+        "id errors download_rate upload_rate upload_rate_unit latency server_name "
```

Comment:
This looks like it is missing `download_rate_unit`

Reply:
:man_facepalming: thanks, fixed